### ISSUE TITLE
Import Asset went to Users

### ISSUE THREAD

**Issue by GRWLX** [grokability/snipe-it#11260]
### Debug mode

- [X] I have enabled debug mode
- [X] I have read [checked the Common Issues page](https://snipe-it.readme.io/docs/common-issues)

### Describe the bug

Import Asset went to Users
Docker

Barcode:,Item:,Number:,comments:,Send:,item photo:,operational:,User ID:,location Barcode,Url Path to Image
0000000000,Netgear prosafe gs108t 8 Port switch,1,Normal,234234234185656.jpg,No,John Lemmy,
0000000001,Netgear prosafe gs108t power supply,1,Normal,234234234185745.jpg,No,John Lemmy,
0000000002,3 and 1/2 by 5 aluminum circuit board enclosure black,1,Normal,234234234185846.jpg,No,John Lemmy,0000000061,
0000000003,USBC nvme M2 external SSD hard drive,1,Normal,234234234190447.jpg,No,John Lemmy,
0000000004,garage Genie main board,1,Normal,234234234185418.jpg,No,John Lemmy,0000000061,
0000000005,Genie power wall converter,1,Normal,234234234185505.jpg,No,John Lemmy,0000000061,
0000000006,okcell 9 volt rechargeable battery,1,Normal,234234234184137.jpg,No,John Lemmy,
0000000007,ezonics multi SD media card reader USB 1.0 with ribbon,1,Normal,234234234185217.jpg,No,John Lemmy,
0000000008,Amway USB 3 7 port with switch self powered,1,Normal,234234234193214.jpg,No,John Lemmy,
0000000009,Pittsburgh Pro 34 piece Precision screwdriver set,1,Normal,234234234185554.jpg,No,John Lemmy,0000000060,

### Reproduction steps

1.Import csv 
2. choose assest
3. goes to user 
...

### Expected behavior

asset import to assets

### Screenshots

![Screenshot 2022-06-06 193957](https://user-images.githubusercontent.com/44076838/172277787-5afc2902-81ee-472c-9260-a273cc3c69ff.png)

### Snipe-IT Version

Snipe-IT versionv6.0.2 build 7961 (g5bc5e9f10)

### Operating System

Ubuntu

### Web Server

Docker

### PHP Version

PHP Version7.4.3

**Comment by snipe** [comment]
Did you pick Users as the import type? Folks use the importer all the time and this is the first I'm hearing about this (and we haven't really changed much in the importer in a while.)

**Comment by GRWLX** [comment]
Picked asset tried three times.

**Comment by snipe** [comment]
@bzeus - can you reproduce this at all? I can't.

**Comment by GRWLX** [comment]
I just reproduced it. Let me try the sample data I gave in the ticket. I also have to delete the users and purge the data.

**Comment by GRWLX** [comment]
![Untitled](https://user-images.githubusercontent.com/44076838/172292882-845d610b-e525-4b8a-9142-12ea2215a839.jpg)


Using the sample data and the image submit for It goes to "People" section no where else

**Comment by snipe** [comment]
@GRWLX I understand that you can reproduce it. @bzeus is our QA engineer. I need her to reproduce it in our environment so I know where to start to troubleshoot this, since (so far) you're the only person reporting this.

**Comment by snipe** [comment]
@GRWLX can you zip up that sample data CSV so we can try using it the way you're using it? Sometimes weird mime types and other stuff can obfuscate the "real" issue, so the closer we can get to reproducing your setup, the better. (GH won't allow csv file IIRC, but does allow zip attachments.)

**Comment by GRWLX** [comment]
[SNIPEITSMALL.txt](https://github.com/snipe/snipe-it/files/8850127/SNIPEITSMALL.txt)

**Comment by nhunterwinsupplyinc** [comment]
Just wondering

**Comment by snipe** [comment]
Hi there - I know this is a pretty old issue, but I wanted to check and see if you ever got this resolved? If so, can you please close the ticket? If not, can you update us on what version of Snipe-IT you're currently using, so we can see if it's still an issue on latest?

:) Thanks!

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)

**Comment by GRWLX** [comment]
Well the test file is there.

**Comment by snipe** [comment]
Yes, but the original ticket was for Snipe-IT versionv 6.0.2, which is from 3 years ago. We cannot reproduce this on the current version.
